src/dotnet/Rider.Plugins.EfCore/EfCoreHelper.cs

```diff
@@ -1,33 +1,53 @@
     {
-        private static readonly string EfCoreDesignNugetId = "Microsoft.EntityFrameworkCore.Design";
-        private static readonly string EfCoreToolsNugetId = "Microsoft.EntityFrameworkCore.Tools";
+        private const string EfCoreDesignNugetId = "Microsoft.EntityFrameworkCore.Design";
+        private const string EfCoreToolsNugetId = "Microsoft.EntityFrameworkCore.Tools";
 
-        public static IEnumerable<IProject> GetSupportedMigrationProjects(ISolution solution) =>
-            GetSupportedDotnetProjects(solution)
+        private const string Net5Target = "net5.0";
+        private const string Net6Target = "net6.0";
+        private const string NetCore31Target = ".NETCoreApp,Version=v3.1";
+        private const string NetStandard21Target = ".NETStandard,Version=v2.1";
+
+        public static IEnumerable<IProject> GetSupportedMigrationProjects(ISolution solution)
+        {
+            var supportedMigrationProjects = GetSupportedDotnetProjects(solution)
                 .Where(project => project.ProjectFileLocation.ExtensionNoDot == "csproj");
 
-        public static IEnumerable<IProject> GetSupportedStartupProjects(ISolution solution) =>
-            GetSupportedDotnetProjects(solution)
+            return supportedMigrationProjects;
+        }
+
+        public static IEnumerable<IProject> GetSupportedStartupProjects(ISolution solution)
+        {
+            var supportedStartupProjects = GetSupportedDotnetProjects(solution)
+                .Where(project => project.TargetFrameworkIds.All(IsNotNetStandard))
                 .Where(project => project.GetInstalledPackage(EfCoreToolsNugetId) != default
                                   || project.GetInstalledPackage(EfCoreDesignNugetId) != default);
 
-        private static IEnumerable<IProject> GetSupportedDotnetProjects(ISolution solution) =>
-            solution
-                .GetAllProjects()
-                .Where(project => project.TargetFrameworkIds
-                    .Any(IsTargetFrameworkSupported));
+            return supportedStartupProjects;
+        }
+
+        private static IEnumerable<IProject> GetSupportedDotnetProjects(ISolution solution)
+        {
+            var supportedDotnetProjects = solution.GetAllProjects()
+                .Where(project => project.TargetFrameworkIds.Any(IsTargetFrameworkSupported));
+
+            return supportedDotnetProjects;
+        }
 
         private static bool IsTargetFrameworkSupported(TargetFrameworkId targetFrameworkId) =>
-            targetFrameworkId.UniqueString == "net5.0"
-            || targetFrameworkId.UniqueString == "net6.0";
+            targetFrameworkId.UniqueString == Net5Target
+            || targetFrameworkId.UniqueString == Net6Target
+            || targetFrameworkId.UniqueString == NetCore31Target
+            || targetFrameworkId.UniqueString == NetStandard21Target;
+
+        private static bool IsNotNetStandard(TargetFrameworkId targetFrameworkId) =>
```

Comment:
I believe that is better to make separate methods for startup projects and migrations projects. Something like `IsStartupTargetFramework` and `IsMigrationsTargetFramework`. It will be cleaner and self-documenting